components/03-organisms/hero-header/hero-header.js

```diff
@@ -0,0 +1,96 @@
+document.addEventListener('alpine:init', () => {
+  Alpine.data('heroHeader', () => ({
+    videoIsPlaying: false,
+    userHasManuallyPaused: false,
+    percentageCompleted: 0,
+    init() {
+      const video = this.$refs['hero-header--video'];
+
+      if (!_.isUndefined(video)) {
+        this.initVideo(video);
+      }
+    },
+    initVideo(video) {
+      this.$watch('videoIsPlaying', (value) => {
+        if (this.videoIsPlaying) {
+          this.$refs['hero-header--video'].play();
```

Comment:
```suggestion
          video.play();
```

My fault, but we are passing `video` into the function so we can use that rather than go via `$refs`